app/views/course.njk

```diff
@@ -93,17 +93,74 @@
       <section class="govuk-!-margin-bottom-8">
         <h2 class="govuk-heading-l" id="entry-requirements">Entry requirements</h2>
 
-        {% set qualificationsAdviceHtml %}{% include "_includes/course/qualifications.md" %}{% endset %}
-        {{ appAdvice({
-          title: "The qualifications you need",
-          html: qualificationsAdviceHtml | trim | markdown
-        }) }}
+        <h3 class="govuk-heading-m" id="required-qualifications">Qualifications needed</h3>
+
+        <p class="govuk-body">
+          {% if course.requirements.degree.minimumClass == "21" %}
+            An undergraduate degree at class 2:1 or above, or equivalent.
+          {% elif course.requirements.degree.minimumClass == "22" %}
+            An undergraduate degree at class 2:2 or above, or equivalent.
+          {% elif course.requirements.degree.minimumClass == "3" %}
+            An undergraduate degree, or equivalent. This should be an honours degree (3rd), or equivalent.
```

Comment:
`3rd` or `third`?

```suggestion
            An undergraduate degree, or equivalent. This should be an honours degree (third), or equivalent.
```

Reply:
Good question! Or even "Third"?

Found examples of both on Find:

> A minimum degree classification 2:2 is preferable, however in exceptional circumstances a Third Class degree may be considered.
https://www.find-postgraduate-teacher-training.service.gov.uk/course/T11/2ZM3#section-entry

> 2:2 degree or above. (NB: A Masters qualification can uplift a 3rd Class Honours degree to a 2:2 to meet our entry requirements)
https://www.find-postgraduate-teacher-training.service.gov.uk/course/2EY/33HW#section-entry

Reply:
`3rd` looks weird to me. Wikipedia says `Third`: https://en.wikipedia.org/wiki/British_undergraduate_degree_classification#Third_Class_honours

Reply:
Yeah, "Third" with an uppercase initial might be the least ambiguous.

Although possibly it needs to be clearer that "honours degree" means Third Class or above?

Reply:
Updated in b7284a49e8f47799aae4cdc4ab35d037e3d74212.

I'll make the same change to the preview in Publish.